packages/cli/src/cmds/compare-report/MarkdownReport.ts

```diff
@@ -106,6 +106,11 @@ export default class MarkdownReport implements Report {
         changeReport.apiDiff.breakingDifferences?.length || 0;
       changeReport.apiDiff.nonBreakingDifferenceCount =
         changeReport.apiDiff.nonBreakingDifferences?.length || 0;
+      Object.keys(changeReport.findingDiff).forEach((key) => {
+        changeReport.findingDiff[key].forEach(
+          (finding: Finding) => ((finding as any).appmap = finding.appMapFile)
+        );
+      });
```

Comment:
@kgilpin I'm not sure why you removed this code but it caused a test failure so I put it back in.